<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Webhook</class>
 <widget class="QMainWindow" name="Webhook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>521</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>432</width>
    <height>521</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>432</width>
    <height>521</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Discord Webhooks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>471</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>461</width>
      <height>471</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>WebHook URL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="webhookInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Avatar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="avatarInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>441</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Content</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="content">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Embeds</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addEmbedButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editEmbedButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteEmbedButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>380</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchFileButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>430</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QListWidget" name="embedsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileDirInput">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
